### DISCUSSION TITLE
How do we run an arbitrary command (not in package.json) in every package?

### DISCUSSION THREAD

**Discussion by trusktr** [lerna/lerna#3864] [Q&A] [answered]
For example, I want to run `rm -rf something` in every package, and this is not a script that each package has.

**Comment by fahslaj** [comment, marked as answer]
Check out [Lerna exec](https://github.com/lerna/lerna/tree/main/libs/commands/exec#lerna-exec). It does exactly what you're looking for.

```
$ lerna exec -- rm -rf something
```
